dart/lib/src/protocol/sentry_device.dart

```diff
@@ -323,7 +328,7 @@ class SentryDevice {
         externalStorageSize: externalStorageSize,
         externalFreeStorage: externalFreeStorage,
         bootTime: bootTime,
-        timezone: timezone,
+        timezone: timezone, // TODO(denrase) Language not cloned?
```

Comment:
@ueman Did we leave out the `language`, `screenHeightPixels` and `screenWidthPixels` on purpose in the `clone` method? (Relates to #465)

Reply:
I don't think so. Got it broken by the merge?

Reply:
It could also very well be that I have somehow missed it

Reply:
kk, i will update this in this pr